### PR TITLE
Abb 24

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Abb 24 new and minor corrections in few others.

### CODE REVIEW

File: ParisBNF/abb/BNFabb24.xml [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+                                <locus target="#293v"/>
+                                <desc>A small drawing of two birds picking berries</desc>
+                            </decoNote>
+                            </decoDesc>
+                        <additions>
+                            
+                            <list>
+                                  <item xml:id="a1">
+                                    <locus from="4ra" to="4rb"/>
+                                      <desc type="GuestText" corresp="LIT5549TribesApostles">The Discourse of Dionysius on the Tribes of the Apostles.</desc>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb24.xml:168]: Indeed! Corrected.

File: ParisBNF/abb/BNFabb105.xml [hunk 2]
```diff
@@ -834,13 +834,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                           <list>
                           <item xml:id="a1">
                             <locus from="1v" to="3r"></locus>
-                             <desc type="GuestText" corresp="LIT6011FeastsList" cert="medium"/>
+                            <desc type="GuestText"><title type="complete" ref="LIT6011FeastsList" cert="medium"></title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb105.xml:837]: `<title type="complete" ref="LIT6011FeastsList" cert="medium">List of Feasts</title>`

File: ParisBNF/abb/BNFabb111.xml [hunk 3]
```diff
@@ -474,7 +474,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                                   <item xml:id="a4">
                                     <locus from="102va" to="102vb"></locus>
-                                     <desc type="GuestText" corresp="LIT3856KingList">Kings list listing
+                                    <desc type="GuestText"><title type="complete" ref="LIT3856KingList"/>Kings list listing
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb111.xml:477]: `<title type="complete" ref="LIT3856KingList">Kings list</title>`

File: ParisBNF/abb/BNFabb24.xml [hunk 4]
```diff
@@ -165,7 +165,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             <list>
                                   <item xml:id="a1">
                                     <locus from="4ra" to="4rb"/>
-                                      <desc type="GuestText" corresp="LIT5549TribesApostles">The Discourse of Dionysius on the Tribes of the Apostles.</desc>   
+                                      <desc type="GuestText"><title type="complete" ref="LIT5549TribesApostles"/>The Discourse of Dionysius on the Tribes of the Apostles.</desc>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb24.xml:168]: `<title type="complete" ref="LIT5549TribesApostles">The Discourse of Dionysius on the Tribes of the Apostles</title>.`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb24.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb24.xml]: ok, hope I corrected it rightly.

**Comment by eu-genia** [comment]
please resolve conflicts

**Review by eu-genia — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**